.gitignore

```diff
@@ -25,3 +25,6 @@ replay_pid*
 
 # Build Files
 target
+
+# Ballerina
```

Comment:
Let's add the reason as a comment.
`Latest versions of dependencies will be used`